packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -29,6 +28,7 @@ import {
 	type User,
 	type WebhookEditMessageOptions
 } from 'discord.js';
+import { isJSONEncodable } from '@discordjs/builders';
```

Comment:
```suggestion
import { isJSONEncodable } from '@discordjs/util';
```

Oh @imranbarbhuiya is right. This is a function from util, not builders.